sasdata/fair_database/data/models.py

```diff
@@ -59,8 +57,9 @@ class DataSet(Data):
     )
 
     # data contents - maybe ManyToManyField
-    data_contents = models.ManyToManyField("Quantity", through="LabeledQuantity")
+    data_contents = models.ManyToManyField("Quantity")
```

Comment:
Maybe I am missing something, but to reduce complexity, I would suggest starting with `data_contents` as a `OneToManyField`. We can reduce storage by allowing a quantity to belong to multiple data sets, but I think it will be cleaner and simpler to manage if every data set has its own data contents.